pandas/plotting/_core.py

```diff
@@ -308,10 +308,24 @@ def _setup_subplots(self):
 
         axes = _flatten(axes)
 
-        if self.logx or self.loglog:
+        valid_log = {False, True, 'sym', None}
```

Comment:
do we explciity list the log parameter in the doc-string? if so need to add a versionadded

Reply:
thanks, @jreback  just updated docstring!